Makefile

```diff
@@ -214,7 +214,7 @@ all-local: $(ALL) ## Build all artifacts in the build tree
 $(ISO_ROOT)/bootstrap.sh: scripts/bootstrap.sh.in $(ISO_ROOT)/product.txt
```

Comment:
If this PR lands, then I want the whole `make` infrastructure to be removed in one go, so we never have two systems in `development/*` at the same time.

However.

I think it may make sense to have some `PHONY` targets in a `Makefile` so `make`, `make vagrantup` etc. still 'work'.

Reply:
Yes, agreed.

We plan to remove it in one shot, but we need to update the CI to rely on doit instead of Make so I have to keep it for now.

Reply:
As far as I'm concerned it's not even 'transient' ;-)

Reply:
But then if I follow your other comment (using `tox`) we will have `make` that calls `tox` that calls `doit`: why so many levels?

For `tox` I get it: it handles the `venv` business, but what's the added value of the make layer/indirection?

Reply:
`make -n4` is quite a bit faster to type than `tox -e doit -- -n 4` (or whatever target name we choose etc).

Reply:
Indeed, so it may be interesting indeed.

Reply:
Forwarding command line argument from `make` to underlying command is either hackish or more verbose, but I found a clean way to do that (handle the venv stuff automatically and run doit transparently).